README.md

```diff
@@ -1,40 +1,33 @@
-GitHub Actions Das Keyboard Q Applet
-===================================
+# GitHub Actions Das Keyboard Q Applet
```

Comment:
Please also update README_END_USER to reflect the fact that it's only on one key